internal/buildscripts/packaging/fpm/rpm/build.sh

```diff
@@ -60,7 +60,6 @@ sudo fpm -s dir -t rpm -n "$PKG_NAME" -v "$VERSION" -f -p "$OUTPUT_DIR" \
     --architecture "$ARCH" \
     --rpm-summary "$PKG_DESCRIPTION" \
     --rpm-use-file-permissions \
-    --depends libcap \
```

Comment:
suse 12.3 was eol 6 years ago: https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/message/NKD3KKIPZXMMHXE7GL7MK3Z23XPUPZBA/.  Is it ok if we'd only support newer versions?  This seems like an undesirable workaround to have to adopt as you have (though seems like the way to go otherwise).

Reply:
I think that opensuse support is different from SLES.  According to https://www.suse.com/lifecycle/, general support for SLES 12 SP5 doesn't end until Oct 31 2024, and LTSS ends Oct 31 2027.